.github/workflows/debian9.yml

```diff
@@ -37,31 +40,37 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-# I386: actions/checkout@V2,  actions/cache@V2 actions/cache@V2 
-#       give error about OCI container [https://github.com/actions/checkout/issues/334]
-#                                      [https://github.com/actions/runner/issues/1011]
         env:
-          - CPU: i386
+          - CC: gcc
+            CXX: g++
+            CPU: i386
+            IMAGE: "i386/debian:9"
+          - CC: clang
+            CXX: clang++
+            CPU: i386
             IMAGE: "i386/debian:9"
     env: ${{ matrix.env }}
     steps:
       - name: Install prerequisites
         run: |
           apt update
           apt -y install git sudo
-      
+
+# I386: actions/checkout@V2,  actions/cache@V2 actions/cache@V2
+#       give error about OCI container [https://github.com/actions/checkout/issues/334]
+#                                      [https://github.com/actions/runner/issues/1011]
       - name: Check out repository
         uses: actions/checkout@v1
         with:
-          fetch-depth: 1
+          fetch-depth: 10
```

Comment:
Is there a reason to fetch 10 commits?

Reply:
I can't find the comment but @maxirmx commented before that on debian:9, fetch-depth of 1 will cause actions/checkout@v1 to fail. A fetch-depth of 10 worked.